services/galley/src/Galley/API/Update.hs

```diff
@@ -890,7 +892,17 @@ withValidOtrBroadcastRecipients ::
   ([(Member, ClientId, Text)] -> Galley ()) ->
   Galley OtrResult
 withValidOtrBroadcastRecipients usr clt rcps val now go = Teams.withBindingTeam usr $ \tid -> do
-  tMembers <- fmap (view userId) <$> Data.teamMembersUnsafeForLargeTeams tid
+  limit <- fromIntegral . fromRange <$> truncationLimit
+  -- If we are going to fan this out to more than limit, we want to fail early
+  unless ((Map.size $ userClientMap (otrRecipientsMap rcps)) <= limit)
+    $ throwM
+    $ broadcastLimitExceeded limit
```

Comment:
Don't forget the swagger docs.

Reply:
https://github.com/wireapp/wire-server/commit/3a0460a7e8bc521e891c95ff93b87f0df086c47c - although this is not static so we have to remove some info from it.